seed/static/seed/partials/analyses_table.html

```diff
@@ -0,0 +1,43 @@
+<div class="section_header_container">
+    <div class="section_header fixed_height_short">
+        <div class="left section_action_container">
+            <h2><i class="fa fa-user"></i> {$:: 'Analyses' | translate $}</h2>
+        </div>
+    </div>
+</div>
+    <div class="section_content_container">
+    <div class="section_content">
+        <div class="table_list_container">
+            <table id="analyses-table" class="table table-striped table_highlight_first">
+                <thead>
+                    <tr>
+                        <th translate>Analysis Name (User Defined)</th>
+                        <th translate>Actions</th>
+                        <th translate>Number of Properties</th>
+                        <th translate>Type</th>
+                        <th translate>Run Status</th>
+                        <th translate>Run Date</th>
+                        <th translate>Run Duration</th>
+                        <th translate>Run Author</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr ng-repeat="analysis in analyses  | filter:filter_params:strict">
+                        <td>{$:: analysis.name $}</td>
+                        <td>
+                            <i class="glyphicon glyphicon-pencil" aria-hidden="true"></i>
+                            <i class="glyphicon glyphicon-trash" aria-hidden="true"></i>
+                        </td>
+                        <td>{$:: analysis.number_of_analysis_property_views $}</td>
+                        <td>{$:: analysis.service $}</td>
+                        <td class="analysis-status {$:: analysis.status.toLowerCase() $}">{$:: analysis.status $}</td>
+                        <td>{$:: analysis.start_time | date : 'MM-dd-yyyy  HH:mm' $}</td>
+                        <td>{$:: analysis | get_run_duration $}</td>
+                        <td>{$:: users | filter : {'user_id':analysis.user} | get_run_author $}</td>
```

Comment:
Yeah, that filter was defined on the analyses_controller.  If we want to have two seperate controllers for this table, this filter should probably be moved to static/seed/js/filters/ and perhaps renamed "get_analysis_run_author" or similar.

Reply:
@aviveiros11 - this seems to be working now, but I'm not exactly sure how? The `get_run_author` filter is still only defined in the analyses_controller, which I don't think is used here... am I missing something?
<img width="943" alt="Screen Shot 2020-12-14 at 2 03 37 PM" src="https://user-images.githubusercontent.com/30608004/102135421-33d89780-3e15-11eb-99d3-40012fa8a1ff.png">

Reply:
@Ryoken or @axelstudios, any ideas here?

Reply:
"Filters allow you to define data transformation, which then can be applied throughout the application without being attached to just one controller or Data type."

It appears the filters are tied to the module rather than the controller it was defined on.  Since there's nothign to be gained keeping them on the controller, we are good to move them into the filters folder.